docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  elasticsearch:
+    image: elasticsearch:latest
+    ports:
+      - "${ELASTICSEARCH_PORT}:${ELASTICSEARCH_PORT}"
+  mongodb:
+    image: mongo:latest
```

Comment:
I was getting a connection error from mongo client. I might be missing something, but possibly the port for mongodb also needs to be forwarded and exposed like it is for elasticsearch?

```
ports:
  - 27017:27017
```